site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+* `ExistingS3LoggingBucket`  
+Specify an existing Amazon S3 bucket instead of creating a new one. The parameter is valid only if `CreateLoggingS3Bucket` is set to `false`.
+
+* `LoggingS3KeyPrefix`  
+A key prefix for the S3 bucket in which logs are stored; for example, "app-name/". By default, no key prefix is specified.
+
+_**PriceClass**_  
+The edge locations that will respond to requests for the CloudFront distribution. The following are valid values:  
+-- "PRICE_CLASS_100": North America and Europe edge locations.  
+-- "PRICE_CLASS_200": North America, Europe, Asia, Middle East, and Africa edge locations.  
+-- "PRICE_CLASS_ALL" (the default): All edge locations (best performance).
+
+_**EnableIpv6**_  
+Specifies whether IPv6 is enabled for the CloudFront distribution. IPv6 is enabled by default. To disable, set this parameter to `false`.
+
+_**MaxHttpVersion**_  
+The maximum HTTP version that users can use to communicate with the CloudFront distribution. The following values are valid:  
+-- "HTTP2" for HTTP version 2 (the default).  
+-- "HTTP1_1" for HTTP version 1.1.
+
+_**WebAclId**_  
+The ARN for the AWS WAF (web application firewall) ACL. For more information, see [Managing and using a web access control list (web ACL)](https://docs.aws.amazon.com/waf/latest/developerguide/web-acl.html) in the  [AWS WAF](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html) developer guide.
+
+## JSON syntax for ASP.NET Core App to AWS App Runner
+
+Use this JSON definition to build an ASP.NET Core application as a container image and deploy it to [AWS App Runner](https://docs.aws.amazon.com/apprunner) (`RecipeId` set to "AspNetAppAppRunner"). If your project doesn't contain a Dockerfile, it will be automatically generated. Recommended if you want to deploy your application as a container image on a fully managed environment.
+
+```
+{
+    "AWSProfile": "string",
+    "AWSRegion": "string",
+    "StackName": "string",
+    "RecipeId": "AspNetAppAppRunner",
+    "OptionSettingsConfig":
+    {
+        "ServiceName": "string",
+        "Port": integer,
+        "StartCommand": "string",
+        "ApplicationIAMRole":
+        {
+            "CreateNew": boolean,
+            "RoleArn": "string"
+        },
+        "ServiceAccessIAMRole":
+        {
+            "CreateNew": boolean,
+            "RoleArn": "string"
+        },
+        "Cpu": "string",
+        "Memory": "string",
+        "EncryptionKmsKey": "string",
+        "HealthCheckProtocol": "string",
+        "HealthCheckPath": "string",
+        "HealthCheckInterval": integer,
+        "HealthCheckTimeout": integer,
+        "HealthCheckHealthyThreshold": integer,
+        "HealthCheckUnhealthyThreshold": integer,
+        "VPCConnector":
+            "CreateNew": boolean,
+            "VpcConnectorId": "string",
+            "VpcId": "string",
+            "Subnets": array,
+            "SecurityGroups": array

```

Comment:
For Subnets and SecurityGroups, the recipe file has: "Type": "List".
Is JSON type "array" correct for this?

Reply:
Yes

Reply:
ACK